keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -151,7 +151,7 @@ capacity.
 To simplify the model, make it backwards compatible and to avoid that it can evolve into something
```

Comment:
Commenting here, but that's in the beginning of the doc:
1. You're missing several checkboxes marked, especially around e2e test for all the new API resources (`ServiceCIDR` and `IPAddress`), implementation history and docs. I checked the test and I don't see tests for those endpoints at all, and the requirements is about all GA endpoints to have conformance tests. I hope this will be handled in the current iteration?

Reply:
ServiceCIDR and IPAddress are the infrastructure that provide IPs to Services, each time a Service is created or deleted both endpoints will be exercised, I will add more text to explain that

Reply:
> and the requirements is about all GA endpoints to have conformance tests

this comment is confusing, how can promote a test to conformance before the feature is GA?

Reply:
so, once the feature moves to GA all the endpoints automatically will be tested by conformance,